spaceship/spec/tunes/app_analytics_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+require 'pp'
+
+describe Spaceship::Tunes::AppAnalytics do
+  before { Spaceship::Tunes.login }
+
+  let(:app) { Spaceship::Application.all.first }
+
+  describe "App Analytics Grabbed Properly" do
+    it "accesses live analytics details" do
+      TunesStubbing.itc_stub_analytics
+
+      analytics = app.analytics
+      units = analytics.app_units
```

Comment:
Can we also add in calls (and `expects`) for all the other analytics methods that are supported so we can get a more complete code coverage? 😇